.script/tests/workbooksValidatorTest/workbooksMetadataValidator.test.ts

```diff
@@ -39,7 +39,7 @@ describe("workbooksValidator", () => {
   });
 
   async function checkInvalid(filePath: string, expectedError: string): Promise<Chai.PromisedAssertion> {
-    expect(IsValidWorkbookMetadata(filePath)).eventually.rejectedWith(Error).and.have.property("name", expectedError);
+    await expect(IsValidWorkbookMetadata(filePath)).eventually.rejectedWith(Error).and.have.property("name", expectedError);
```

Comment:
When there is no await and running this test locally, when a test fails then it shows that it passes but raises a warning "UnhandledPromiseRejectionWarning".
Adding await solves this issue

Reply:
thanks for the catch!